### DISCUSSION TITLE
Welcome to garak Discussions!

### DISCUSSION THREAD

**Discussion by leondz** [NVIDIA/garak#189] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas about LLM security.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 🧑‍🤝‍🧑.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomBarnes** [comment]
Hello.  I'm looking to set up Garak and I'm wondering how I will know if it is working or not.  How do you test Garak to verify that tit is working as expected? Do you use a test LMM that exhibits known bad behavior? Or do you use some other technique?

Thanks,

-Tom
